cylibs/trust/roles/combat_mode.lua

```diff
@@ -29,7 +29,7 @@ function CombatMode.new(action_queue, melee_distance, range_distance, addon_enab
     local self = setmetatable(Role.new(action_queue), CombatMode)
 
     self.action_queue = action_queue
-    self.melee_distance = melee_distance
+    self.melee_distance = 2
```

Comment:
Currently this is a configurable value in `settings.xml`. Should we just update that value instead and do a migration to change the default to 2? Or will the new logic always put the player at maximum melee distance?

Reply:
New logic will always put them at correct max melee with that set to 2. Making it configurable could let users put it higher which will put them out of range

Reply:
How confident are you that this works? I'm a bit nervous about people getting out of range messages when I know just setting it to 2 or 3 will always put you in range.

Reply:
Almost completely. I have yet to see any issues with range using this config. The last time I saw it, there was an issue that I got out of range on the dyna midboss statue, but I changed the -0.1 to -0.2 which seems to resolve it. I suspect that's some wild edge case with the stairs in Jeuno or something, but yeah, this should always put you in melee range now.